docs/api/item.md

```diff
@@ -132,6 +237,14 @@ import InputHighlight from '@site/static/usage/v7/item/theming/input-highlight/i
 
 <InputHighlight />
 
+## Usage Guidelines
```

Comment:
```suggestion
## Guidelines
```

Maybe we should just call this `Guidelines` like [Material Design](https://m3.material.io/components/lists/guidelines) does?